common/djangoapps/student/urls.py

```diff
@@ -45,6 +45,7 @@
 if settings.FEATURES.get('AUTOMATIC_AUTH_FOR_TESTING'):
     urlpatterns += (
         url(r'^auto_auth$', 'auto_auth'),
+        url(r'^update_user$', 'update_user'),
```

Comment:
We are adding a new url pattern here, Will it map automatically on the existing frontend request?